rectools/models/nn/transformers/sasrec.py

```diff
@@ -101,6 +106,14 @@ def _collate_fn_train(
             batch_dict["negatives"] = self.negative_sampler.get_negatives(
                 batch_dict, lowest_id=self.n_item_extra_tokens, highest_id=self.item_id_map.size
             )
+        if self.add_unix_ts:
```

Comment:
in 3 `_collate_fn_...` functions we have very similar pieces of code for this. 
Let's please try to write it in one place and reuse